steis.R

```diff
@@ -68,14 +68,54 @@ if (!is.na(text_terms)) {
   steis_filtered_text <- steis_filtered_categorical
 }
 
+# Neonate Search
+if (is_neopaed == "neonate") {
+  print( "- Running neonate strategy...")
+steis_neonate <- steis_filtered_text |>
+  filter(
+    (patient_age_years < 1)
+    | 
+      (
+        is.na(patient_age_years) &
+          (
+            grepl("(?i)\\bneonat|\\bbaby", paste(type_of_incident_other, clinical_area_other, care_sector_other, sep = "")) |
+              (
+                grepl("(?i)\\bn(?:|\\W)i(?:|\\W)c(?:|\\W)u\\b|\\bn(?:|\\W)n(?:|\\W)u\\b|\\bs(?:|\\W)c(?:|\\W)b(?:|\\W)u\\b|\\bneonat|\\bbaby", paste(care_sector_other, type_of_incident_other, clinical_area_other, description_of_what_happened, immediate_action_taken, key_findings, case_summary, how_will_lessons_be_disseminated_to_interested_parties, sep = "")) &
+                  !grepl("(?i)\\badult|\\bold|\\belderly|\\bgeriat", paste(care_sector_other, clinical_area_other, type_of_incident_other, sep = "")) 
```

Comment:
Can you check this isn't pulling any young person specialties by accident e.g. 'young adult'